src/apiCalls.js

```diff
@@ -9,7 +9,6 @@ export const getSingleCity = (id) => {
 }
 
 export const checkError = (response) => {
-  console.log(response)
```

Comment:
Thanks for removing this line for me!  I realized I'd left it in as soon as I'd pushed the previous branch.